### PR TITLE
Feat/review form

### PR DESCRIPTION
**Pull request by NNaumenko83**
Hi. I have created ReviewForm.  Check it please.

### CODE REVIEW

File: src/shared/AppTextArea/AppTextArea.jsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const AppTextArea = ({
   value,
   onChange,
   placeholder = '',
+  name = '',
```
Comment (4m4rel) [inline src/shared/AppTextArea/AppTextArea.jsx:21]: better to provide it through `defaultProps`

File: src/shared/Buttons/AppButton.jsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ export default AppButton;
 AppButton.propTypes = {
   type: PropTypes.oneOf(['contained', 'text', 'outlined']),
   label: PropTypes.string.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```
Comment (4m4rel) [inline src/shared/Buttons/AppButton.jsx:47]: if prop is not required provide default for it in `defaultProps`

File: src/shared/AppTextArea/AppTextArea.jsx (hunk 1 above)
Comment (NNaumenko83) [reply src/shared/AppTextArea/AppTextArea.jsx]: Hi. I changed it.

File: src/shared/Buttons/AppButton.jsx (hunk 2 above)
Comment (NNaumenko83) [reply src/shared/Buttons/AppButton.jsx:47]: Hi. I changed it.

**Review by 4m4rel — APPROVED** [review]

File: src/shared/Buttons/AppButton.jsx (hunk 2 above)
Comment (kyzym) [reply src/shared/Buttons/AppButton.jsx:47]: type = 'contained', variant={type}, type={type}
Something a little strange.

Comment (NikolayVerstak) [reply src/shared/Buttons/AppButton.jsx:47]: Guys, I fixed it and already pushed to main. My mistake

File: src/components/ReviewForm/ReviewForm.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          maxRows={5}
+          placeholder="Leave your feedback"
+          value={review}
+          onChange={handleTextareaChange}
+          maxLength={400}
+          name="review"
+        />
+      </label>
+      <ButtonWrapper>
+        <AppButton type="contained" label="Send" />
```
Comment (NikolayVerstak) [inline src/components/ReviewForm/ReviewForm.jsx:53]: I fixed the bug with AppButton props. So you can change it to the correct one <AppButton variant="contained" label="Send" />

Comment (NNaumenko83) [reply src/components/ReviewForm/ReviewForm.jsx]: Hi. I changed it. Could you approve my PR? )

**Review by kyzym — APPROVED** [review]

**Review by NikolayVerstak — APPROVED** [review]

**Pull request merged**